linode/lke/cluster.go

```diff
@@ -363,7 +375,7 @@ func matchPoolsWithSchema(pools []linodego.LKENodePool, declaredPools []interfac
 		declaredPool := declaredPool.(map[string]interface{})
 
 		for key, pool := range poolMap {
-			if pool.Count != declaredPool["count"] || pool.Type != declaredPool["type"] {
```

Comment:
Selecting on count was causing super broken diffs, ideally we should calculate similarity scores for this function.